bach/tests/conftest.py

```diff
@@ -76,7 +76,11 @@ class DB(Enum):
 
 
 @pytest.fixture()
-def pg_engine() -> Engine:
+def pg_engine(request: SubRequest) -> Engine:
+    if request.config.getoption("big_query"):
```

Comment:
we could do
```suggestion
    if DB.POSTGRES not in _ENGINE_CACHE:
```
then it would also keep working if we add, for example, an `--athena` option

Reply:
like it, applied